tasks/__init__.py

```diff
@@ -79,7 +79,7 @@ def test_travis_on_prod(ctx):
     """
     flake(ctx)
     print('>>> Testing modules in "{}" in Chrome'.format('tests'))
```

Comment:
Not related to this PR but looks like `test_travis_on_prod` have a general `print` message which always uses `Chrome` while it is called three times for `Chrome`,  `Firefox` and `Edge`.

See: https://github.com/cos-qa/osf-selenium-tests/blob/e08ebac4c757e698cf32d6092fc4aea72d817f94/.travis.yml#L32-L52